llvm/test/CodeGen/AArch64/shrink-wrap-const-pool-access.mir

```diff
@@ -24,8 +21,11 @@
     br label %common.ret
   }
 ...
-# Following code has a load from constant pool. Accessing constant pool must not
-# be considered as a stack access and hence, shrink wrapping must happen.
+# FIXME: Following code has a load from constant pool. Accessing constant pool
+# must not be considered as a stack access and hence, shrink wrapping must
+# happen.
+# CHECK-NOT: savePoint
+# CHECK-NOT: restorePoint
```

Comment:
Please add a CHECK line that match on the constant pool load.

The CHECK-NOT line still don't tell me why this is testing that shrink wrapping happens/doesn't happen.
I don't remember the implementation details, but assuming that when shrink wrapping happens we create a `savePoint` or `restorePoint` basic block, just check for that and say it here.